gcovr/json_generator.py

```diff
@@ -67,7 +68,7 @@ def print_json_summary_report(covdata, output_file, options):
 
     json_dict = {}
     json_dict['current_working_directory'] = options.root
-    json_dict['gcovr/format_version'] = JSON_FORMAT_VERSION
+    json_dict['gcovr/summary_format_version'] = JSON_SUMMARY_FORMAT_VERSION
```

Comment:
What do you think about change:
    json_dict['gcovr/summary_format_version'] = JSON_SUMMARY_FORMAT_VERSION
to:
    json_dict['summary_format_version'] = JSON_SUMMARY_FORMAT_VERSION

without `gcovr` prefix? As it is mentioned in the [User Guide](https://github.com/gcovr/gcovr/blob/master/doc/source/guide.rst#id18) `gcovr/` was used to differentiate between GCOV keys and gcovr keys. This summary doesn't base on some external format, so there is nothing to differentiate.

Reply:
@CezaryGapinski `json` is notorious about not telling what this payload is. I would rather keep the `gcovr` in place instead of ambiguous `summary_format_version`.